### PR TITLE
Add --kubelet-object option to helm chart

### LINKED ISSUE #273: Helm chart is missing the --kubelet-object option
**What did you do?**

I installed the operator as explained [here](https://github.com/coreos/prometheus-operator/tree/92b5a896d2265f89caa1f87240291ba623ea8a99/helm/prometheus-operator), disabling rbac because I am running on K8s 1.5.

**What did you expect to see?**

I expected to see endpoints for the kubelets in the kube-system namespace, to be able to monitor them.

**Environment**

* Kubernetes version information:

```
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"5", GitVersion:"v1.5.4", GitCommit:"7243c69eb523aa4377bce883e7c0dd76b84709a1", GitTreeState:"clean", BuildDate:"2017-03-07T23:53:09Z", GoVersion:"go1.7.4", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"5", GitVersion:"v1.5.4", GitCommit:"7243c69eb523aa4377bce883e7c0dd76b84709a1", GitTreeState:"clean", BuildDate:"2017-03-07T23:34:32Z", GoVersion:"go1.7.4", Compiler:"gc", Platform:"linux/amd64"}
```

* Kubernetes cluster kind: 

	Cluster created on AWS with kops.

**Note**

The chart is probably missing the ``--kubelet-object`` option [here](https://github.com/coreos/prometheus-operator/blob/92b5a896d2265f89caa1f87240291ba623ea8a99/helm/prometheus-operator/templates/deployment.yaml#L25).

### PR DESCRIPTION
**Pull request by MrTrustor**
Fixes #273

### CODE REVIEW

**Comment by fabxc** [comment]
👍

**Comment by mgoodness** [comment]
Do we want a conditional around this? Enabled or disabled by default?

File: helm/prometheus-operator/templates/deployment.yaml [hunk 1]
```diff
@@ -22,6 +22,7 @@ spec:
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: "{{ .Values.image.pullPolicy }}"
           args:
+            - --kubelet-object={{ .Values.kubeletObject }}
```
Comment (mgoodness) [inline helm/prometheus-operator/templates/deployment.yaml]: Do we want a conditional around this? Enabled or disabled by default?

Comment (MrTrustor) [inline helm/prometheus-operator/templates/deployment.yaml]: Yes, probably. I am using the operator to monitor kubernetes itself so I didn't think of that. I will add the conditional.
I am going to enable it by default, unless someone is against it: if the option is here but we don't need it, it's not a big deal.

**Comment by brancz** [comment]
lgtm

**Pull request merged**